### PR TITLE
refactor:made minor changes

### PR DESCRIPTION
**Pull request by shivakarinigi**
Moved the query parameters validation just after the event.

### CODE REVIEW

File: workflow/project/delete_usecase.js
Comment (Sohailshah20) [inline workflow/project/delete_usecase.js:1]: Along with null also check for non-empty string

File: workflow/project/getProjectsUsecases.js
Comment (Sohailshah20) [inline workflow/project/getProjectsUsecases.js:1]: check for non-empty string

File: workflow/task/getTaskDetails.js
Comment (Sohailshah20) [inline workflow/task/getTaskDetails.js:1]: check for non empty string

**Review by Sohailshah20 — CHANGES_REQUESTED** [review]

File: workflow/task/getTaskDetails.js
Comment (shivakarinigi) [reply workflow/task/getTaskDetails.js:1]: okay

File: workflow/project/delete_usecase.js
Comment (shivakarinigi) [reply workflow/project/delete_usecase.js:1]: completed

File: workflow/project/getProjectsUsecases.js
Comment (shivakarinigi) [reply workflow/project/getProjectsUsecases.js:1]: completed

File: workflow/task/getTaskDetails.js
Comment (shivakarinigi) [reply workflow/task/getTaskDetails.js:1]: completed

**Review by Sohailshah20 — APPROVED** [review]

**Pull request merged**